ios/src/platform_ios.mm

```diff
@@ -1,15 +1,21 @@
 #ifdef PLATFORM_IOS
 
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
 #import <utility>
 #import <cstdio>
 #import <cstdarg>
 #import <fstream>
 #import <regex>
 
+#include <iostream>
```

Comment:
You can probably swap all of these over to using import rather than include - it's safer. Also, it looks like you're importing <fstream> twice?